Assets/Runtime/ScreenTransitions/SimpleFadeTransition.cs

```diff
@@ -52,22 +58,36 @@ public class SimpleFadeTransition : ATransitionComponent
             shouldAnimate = true;
         }
 
-        public override void Stop(Transform target) {
+        public override void Stop(Transform target)
+        {
             shouldAnimate = false;
         }
 
-        private void Update() {
-            if (!shouldAnimate) {
+        private void Update()
+        {
+            if (!shouldAnimate || canvasGroup == null)
```

Comment:
is not `canvasGroup == null` only if `shouldAnimate == false`?

Reply:
agree